File: src/cdva/tests/fixtures/elixir/multibyte.ex
defmodule Greeter do
  def greet do
    "こんにちは"
  end
end

defmodule GreeterChecks do
  use ExUnit.Case

  test "日本語で挨拶する" do
    assert Greeter.greet() == "こんにちは"
  end
end

File: src/cdva/tests/fixtures/swift/multibyte.swift
import XCTest

struct Greeter {
    func greet() -> String {
        return "こんにちは"
    }
}

final class GreeterChecks: XCTestCase {
    func testGreetsInJapanese() {
        // 挨拶を確かめる
        XCTAssertEqual(Greeter().greet(), "こんにちは")
    }
}
